src/CodeStyle/Core/CodeFixes/Host/Mef/CodeStyleHostLanguageServices.MefHostExportProvider.cs

```diff
@@ -43,8 +43,8 @@ private static CodeStyleHostLanguageServices Create(HostLanguageServices hostLan
 
         public override string Language => _hostLanguageServices.Language;
 
-        [return: MaybeNull]
-        public override TLanguageService GetService<TLanguageService>()
+        public override TLanguageService? GetService<TLanguageService>()
+            where TLanguageService : default
```

Comment:
just curious, how is the experience making this migration from `[MaybeNull] T` to `T?`? Is it kinda at a point where if you haven't read recent blog posts or LDM notes you'd be lost?

Reply:
Not nearly as bad as enabling NRT for the first time. The `default` constraint is the only place I had trouble. We desperately need a code fix for this particular error or we're going to be fielding a ton of "WTF" questions.